MAINTAINERS.md

```diff
@@ -17,76 +17,82 @@ The following is the answer to that question **for Astro Maintainers**, so that
 
-- Consult the [Internationalization Guide](/src/i18n/README.md) to familiarize yourself with the transators' process
+- Consult the [Internationalization Guide](/src/i18n/README.md) to familiarize yourself with the tranlsators' process.
 - If you see "LGTM" in a message within the PR, that means at least one other native speaker has approved the translation, and the PR can be immediately merged! 🥳
-- If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides located in their language folder within `/src/i18n/`
-- If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PR’s Netlify deploy preview to verify that nothing is visually out of place.
+- If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides located in their language folder within `/src/i18n/`.
+- If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PR’s Netlify deploy preview to verify that nothing is visually out of place. While we always prefer to have a review from a native speaker of the language, having translated docs with some errors is usually better than having no docs at all.
 
 
 ## Submitting your own PRs
 
 This section is much shorter now that we are protecting the `main` branch of the Astro repos! 😅
 
-Maintainers who submit PRs are expected to **merge their own PRs**, but only **after receiving an LGTM**. 
+If your PR depends on a core PR being merged first, please **mark your PR as a draft** so that it does not get merged first. Please also link to the other PR so that we can easily check on its status when we are doing PR reviews.
 
-Typically, this will be an approval from another maintainer, often the Docs Lead or Docs Infrastructure Lead. But, in the case of minor PRs (e.g. typo fixes, broken links etc.), any second LGTM is acceptable.
+Other maintainers who leave review comments will mention you by name so that you will receive a GitHub notification. This is our default way of alerting you of activity on your PR. **If this is not a good way to get your attention, then please state in the PR whether you would like to be pinged on Discord.**
 
+Maintainers who submit PRs are expected to **merge their own PRs**, but only **after receiving an LGTM**.
 
+Typically, this will be an approval from another maintainer, often the Docs Lead or Docs Infrastructure Lead. But, in the case of minor PRs (e.g. typo fixes, broken links etc.), any second LGTM is acceptable.
```

Comment:
```suggestion
Typically, this will be an approval from another maintainer, often the Docs Lead or Docs Infrastructure Lead. But, in the case of minor PRs (e.g. typo fixes, broken links etc.), any second LGTM is acceptable.

Lastly, we have a "DocsBot" that posts congratulation messages in the #docs Discord channel. **Our bot also acknowledges co-authors!** During "Squash and Merge," if you would like to acknowledge any reviewer, please type in the exact phrase `Co-authored-by:` and include any missing reviewers or contributors whom you'd like to acknowedge!

TODO: List the steps to get the proper name to use in commit message.
```